package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "airbnb-prop-types",
-  "version": "2.5.4",
+  "version": "2.5.5",
```

Comment:
please revert this; versions should never be bumped in PRs; only ever in a commit directly to master.